traffic_ops/build/build_rpm.sh

```diff
@@ -76,25 +76,15 @@ function initBuildArea() {
                 { echo "Could not build atstccfg binary"; exit 1; }
 	popd
 
-	rsync -av etc install "$to_dest"/ || \
-		 { echo "Could not copy to $to_dest: $?"; exit 1; }
-	rsync -av app/{bin,conf,cpanfile,db,lib,public,script,templates} "$to_dest"/app/ || \
-		 { echo "Could not copy to $to_dest/app: $?"; exit 1; }
-	tar -czvf "$to_dest".tgz -C "$RPMBUILD"/SOURCES $(basename "$to_dest") || \
-		 { echo "Could not create tar archive $to_dest.tgz: $?"; exit 1; }
-	cp "$TO_DIR"/build/*.spec "$RPMBUILD"/SPECS/. || \
+	rsync -av doc etc install "$dest"/ || \
```

Comment:
`doc` was removed back in 5b035a9d0d and re-adding it breaks the Traffic Ops build script.